lib/ci-stack.ts

```diff
@@ -31,12 +31,14 @@ export interface CIStackProps extends StackProps {
   runWithOidc?: boolean;
   /** Additional verification during deployment and resource startup. */
   strictMode?: boolean;
+  /** Environment the stack is ebing deployed on */
```

Comment:
Typo: being. Also I think this should be reframed as `Account to deploy your ECR Assets on`

Reply:
It is not the account, but the env. Account number is a different property. I'll fix the typo